static/samples/k8s/pxc/portworx-node-dashboard.json

```
@@ -1,1529 +1,1530 @@
+          "refId": "A",
+          "resultFormat": "time_series",
+          "select": [
+            [
+              {
+                "params": [
+                  "gauge"
+                ],
+                "type": "field"
+              },
+              {
+                "params": [],
+                "type": "sum"
+              }
+            ]
+          ],
+          "tags": [
             {
-              "from": "null",
-              "text": "N/A",
-              "to": "null"
+              "key": "cluster",
+              "operator": "=~",
+              "value": "/^$Cluster$/"
             }
-          ],
-          "sparkline": {
-            "fillColor": "rgba(31, 118, 189, 0.18)",
-            "full": true,
-            "lineColor": "rgb(31, 120, 193)",
-            "show": true
-          },
-          "tableColumn": "",
-          "targets": [
+          ]
+        },
+        {
+          "alias": "Cluster Disk Used",
+          "dsType": "influxdb",
+          "expr": "px_cluster_disk_utilized_bytes{cluster=~\"[[Cluster]]\",instance=~\"[[Instances]]\"}",
+          "format": "time_series",
+          "groupBy": [
+            {
+              "params": [
+                "10s"
+              ],
+              "type": "time"
+            },
             {
-              "expr": "min(px_cluster_status_cluster_size{cluster=~\"[[Cluster]]\"}) ",
-              "format": "time_series",
-              "intervalFactor": 1,
-              "legendFormat": "",
-              "refId": "A"
+              "params": [
+                "host"
+              ],
+              "type": "tag"
+            },
+            {
+              "params": [
+                "null"
+              ],
+              "type": "fill"
             }
           ],
-          "thresholds": "",
-          "title": "Members",
-          "type": "singlestat",
-          "valueFontSize": "80%",
-          "valueMaps": [
+          "hide": false,
+          "intervalFactor": 1,
+          "legendFormat": "{{cluster}}{{instance}}",
+          "measurement": "px_cluster_disk_utilized_bytes",
+          "orderByTime": "ASC",
+          "policy": "default",
+          "refId": "B",
+          "resultFormat": "time_series",
+          "select": [
+            [
+              {
+                "params": [
+                  "gauge"
+                ],
+                "type": "field"
+              },
+              {
+                "params": [],
+                "type": "sum"
+              }
+            ]
+          ],
+          "tags": [
             {
-              "op": "=",
-              "text": "N/A",
-              "value": "null"
+              "key": "cluster",
+              "operator": "=~",
+              "value": "/^$Cluster$/"
             }
-          ],
-          "valueName": "current"
+          ]
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "PWX Disk Usage",
```

If possible, we should avoid abbreviating Portworx to protect our brand. If there's just not enough room, I think it should be abbreviated to "PX" instead of "PWX" to remain consistent with the shorthand elsewhere. 
```suggestion
      "title": "Portworx Disk Usage",
```